bin/templates/cordova/lib/build.js

```diff
@@ -266,6 +270,8 @@ module.exports.help = function () {
     console.log('    \'--prepenv\': don\'t build, but copy in build scripts where necessary');
     console.log('    \'--versionCode=#\': Override versionCode for this build. Useful for uploading multiple APKs.');
     console.log('    \'--minSdkVersion=#\': Override minSdkVersion for this build. Useful for uploading multiple APKs.');
+    console.log('    \'--maxSdkVersion=#\': Override maxSdkVersion for this build. Useful for uploading multiple APKs.');
```

Comment:
Remove "Useful for uploading multiple APKs." from `minSdkVersion`, `maxSdkVersion`,  and`targetSdkVersion`.

There's also probably no value in supporting `maxSdkVersion` since they really *really* strongly discourage using it (your app will not be compatible with newer versions of Android)

Reply:
At best, we might only be able to add to the printout `(Not Recommended)`.

We already supported the setting of the `maxSdkVersion` property through `config.xml` and in which sets it in `AndroidManifest.xml`.

This PR moves the `maxSdkVersion` definition into the Gradle's workflow and removes from `AndroidManifest.xml`. 

This does add a new command line option to exposes alternative ways for defining this value other than `config.xml`. I thought it was a missed option because we supported `min` and believe there was no harm to add.

If we want to remove `maxSdkVersion` altogether, it then becomes a `major` change. As for future planning, it would be best if I dont add the new command-line option then and then add warnings that the `maxSdkVersion` setting will be deprecated in next major.

Thoughts and Opinions?